src/praisonai/praisonai/framework_adapters/praisonai_adapter.py

```
@@ -1,106 +1,265 @@
+                # Auto-generate a task
+                task_description = details.get('instructions') or backstory_filled
+                task = PraisonTask(
+                    description=task_description,
+                    expected_output="Complete the assigned task successfully.",
+                    agent=agent,
+                )
+                if task_callback:
+                    task.callback = task_callback
+                tasks.append(task)
+            else:
+                for task_name, task_details in agent_tasks.items():
```

Ruff (0.15.17)

[warning] Loop control variable `task_name` not used within loop body

Rename unused `task_name` to `_task_name`

(B007)